test/uk/gov/hmrc/crsfatcafimanagement/connector/CADXConnectorSpec.scala

```diff
@@ -75,6 +75,34 @@ class CADXConnectorSpec extends SpecBase with WireMockServerHandler with Generat
         }
       }
     }
+    "remove FI" - {
+      "must return status as OK for removal request" in {
+        stubResponse(
+          "/ASMService/v1/FIManagement",
+          OK
+        )
+
+        forAll(arbitrary[RemoveFIDetailsRequest]) {
+          req =>
+            val result = connector.removeFI(req)
+            result.futureValue.status mustBe OK
+        }
+      }
+
+      "must return an error status for failed remove request" in {
+
+        forAll(arbitrary[CreateFIDetailsRequest], errorCodes) {
```

Comment:
`RemoveFIDetailsRequest` ?

Reply:
oops, yes, spose this would also pass this test aha